src/index.js

```diff
@@ -519,6 +525,7 @@ export function stream(source: string, index: number=0): () => SourceLocation {
         case DSTRING_START:
         case TSSTRING_START:
         case TDSTRING_START:
+        case HEREGEXP_START:
           setType(STRING_CONTENT);
```

Comment:
Yeah, my thought was that heregexes would reuse the same interior token types as strings: STRING_CONTENT, INTERPOLATION_START, INTERPOLATION_END, and probably later STRING_PADDING. That should hopefully make it easier to have shared code for dealing with interpolations rather than a different code path for each case.

For example, `///a${b}c///` would look like HEREGEXP_START, STRING_CONTENT, INTERPOLATION_START, IDENTIFIER, INTERPOLATION_END, STRING_CONTENT, HEREGEXP_END.